### DISCUSSION TITLE
Is there any way to specify the `history.txt` file path?

### DISCUSSION THREAD

**Discussion by abehidek** [nushell/nushell#9070] [Q&A] [answered]
The cli allow us to change the config, env-config and plugin-config paths, however it would be good to also change the history of commands path.

**Comment by amtoine** [comment]
can we change `$nu.config-path`, `$nu.env-path` and `$nu.plugin-path` from the REPL? :open_mouth:

**Reply by ghost** [reply]
as far as im aware no
```
Error: nu::parser::name_is_builtin_var

  × `nu` used as variable name.
   ╭─[entry #2:1:1]
 1 │ let nu = ($nu | upsert env-config ~/a.txt)
   ·     ─┬
   ·      ╰── already a builtin variable
   ╰────
  help: 'nu' is the name of a builtin Nushell variable and cannot be used
        as a variable name
```
i think the question was refering to the launch arguments `--config` and `--env-config` (and plugins are changeable from `$env.config` as well as `--plugin-config`

**Reply by amtoine** [reply]
oooooh the `nu` options!
is that it @Jan9103? :yum:

**Reply by amtoine** [reply, marked as answer]
if that's it, yes i do not think we can change that from the CLI with `nu` options, e.g. `nu --history-path`
